modules/geo-layers/src/tile-layer/tile-2d-header.ts

```diff
@@ -44,13 +73,19 @@ export default class Tile2DHeader {
   }
 
   /* eslint-disable max-statements */
-  async _loadData({getData, requestScheduler, onLoad, onError}) {
+  private async _loadData({
+    getData,
+    requestScheduler,
+    onLoad,
+    onError
+  }: TileLoadDataProps): Promise<void> {
     const {x, y, z, bbox} = this;
     const loaderId = this._loaderId;
 
-    this._abortController = new AbortController(); // eslint-disable-line no-undef
+    this._abortController = new AbortController();
     const {signal} = this._abortController;
 
+    // @ts-expect-error (2345) Argument of type '(tile: any) => 1 | -1' is not assignable ...
     const requestToken = await requestScheduler.scheduleRequest(this, tile => {
```

Comment:
This error is caused by wrong type of `GetPriorityFunction` defined [request-scheduler.ts](https://github.com/visgl/loaders.gl/blob/master/modules/loader-utils/src/lib/request-utils/request-scheduler.ts#L5) of `@loaders.gl/loader-utils`.